src/test/commons/com/trzewik/information/producer/interfaces/rest/information/InformationFormsCreation.groovy

```diff
@@ -1,7 +1,19 @@
 package com.trzewik.information.producer.interfaces.rest.information
 
+import com.trzewik.information.producer.domain.information.CarCreation
+import com.trzewik.information.producer.domain.information.InformationCreation
+import com.trzewik.information.producer.domain.information.PersonCreation
+
 trait InformationFormsCreation {
```

Comment:
I also think, that this trait should be refactored.... `InformationCreator` should has `person` and `cars` fields instead of `personCreator` and `carCreators`

Reply:
When you start working on it, please give me call. We will discuss how it should work properly ;)